packages/patterns/package.json

```diff
@@ -2,8 +2,16 @@
   "name": "bigcommerce-design-patterns",
   "description": "Sandbox for Patterns",
   "version": "0.11.1",
-  "main": "./src/index.js",
+  "main": "./src/index.ts",
   "types": "./dist/index.d.ts",
+  "exports": {
```

Comment:
Wondering why was this changed. Was it a recommendation or fixing something you ran into?